pkgs/tools/inputmethods/footswitch/default.nix

```
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, hidapi }:
+
+stdenv.mkDerivation {
+  pname = "footswitch";
+  version = "1.0+20210317.aa0b10f";
```

The [nixpkgs manual requires](https://nixos.org/manual/nixpkgs/stable/#sec-package-naming) the usage of the `unstable-YYYY-MM-DD` format for packaging unreleased software.

```suggestion
  version = "unstable-2021-03-17";
```